library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionViewModel.kt

```diff
@@ -49,14 +49,32 @@ internal class TransactionViewModel(transactionId: Long) : ViewModel() {
     fun encodeUrl(encode: Boolean) {
         mutableEncodeUrl.value = encode
     }
+
+    suspend fun onBodyLineItemClicked(url: String): Action {
+        val request = RepositoryProvider.transaction().getByUrl(url)
```

Comment:
I'm rather hesitant about this functionality. URL by themselves are not unique. `POST`, `PUT`, `GET` and so on can use the same path. Moreover, one endpoint can be invoked multiple times and there will be many instances of it in our database and this just navigates to whichever happens to be latest.